code/game/objects/items/tanks/tank_types.dm

```diff
@@ -176,3 +176,21 @@
 
 /obj/item/tank/internals/emergency_oxygen/double/empty/populate_gas()
 	return
+
+/obj/item/tank/internals/ipc_coolant
```

Comment:
are we ready to make /obj/item/tank/internals/species instead of adding species tanks to every job next time :'(